doc/Quick_Start/README.md

```diff
@@ -171,7 +175,7 @@ https://docs.rakwireless.com/Product-Categories/WisBlock/RAK1910/Overview/#produ
 
 RAK2305 is a WiFi board which is ESP32 module actually, and it can only be connected with RAK5005-O through the IO SLOT.
 
-<center><img src="../../assets/repo/rak2305-top.png" alt="rak2305-top" width="20%"/> <img src="../../assets/repo/rak2305-bottom.png" alt="rak2305-bottom" width="30%" /></center>
+<center><img src="../../assets/repo/rak2305-top.png" alt="rak2305-top" width="30%"/> <img src="../../assets/repo/rak2305-bottom.png" alt="rak2305-bottom" width="30%" /></center>
```

Comment:
First 2 pictures can be enlarged by 20-30%, as they have many details.

Reply:
The problem is if I enlarge them they will not be shown in one row on smaller screens. 
I prefer to keep them smaller.